src/Fabulous.XamarinForms/Views/Controls/Image.fs

```diff
@@ -9,22 +11,35 @@ type IImage =
 module Image =
     let WidgetKey = Widgets.register<Image> ()
 
-    let Source =
-        Attributes.defineBindable<ImageSource> Image.SourceProperty
-
     let Aspect =
         Attributes.defineBindable<Aspect> Image.AspectProperty
 
+    let IsAnimationPlaying =
+        Attributes.defineBindable<bool> Image.IsAnimationPlayingProperty
+
+    let IsOpaque =
+        Attributes.defineBindable<bool> Image.IsOpaqueProperty
+
+    let Source =
+        Attributes.defineAppThemeBindable<string> Image.SourceProperty
+
 [<AutoOpen>]
 module ImageBuilders =
     type Fabulous.XamarinForms.View with
         static member inline Image<'msg>(imageSource: ImageSource, aspect: Aspect) =
             WidgetBuilder<'msg, IImage>(
                 Image.WidgetKey,
-                Image.Source.WithValue(imageSource),
+                (Attributes.defineBindable<ImageSource> Image.SourceProperty)
+                    .WithValue(imageSource),
                 Image.Aspect.WithValue(aspect)
             )
 
+        static member inline Image<'msg>(light: string, ?dark: string) =
```

Comment:
Even though I understand you might want the default `aspect`, I think making it mandatory is actually better to be sure of what will be displayed on screen :)